queries/orderbook/quote_rewards.sql

```diff
@@ -3,8 +3,7 @@ with winning_quotes as (SELECT concat('0x', encode(oq.solver, 'hex')) as solver,
                         FROM trades t
                                INNER JOIN orders o ON order_uid = uid
                                JOIN order_quotes oq ON t.order_uid = oq.order_uid
-                        WHERE o.class = 'market'
-                          AND block_number BETWEEN {{start_block}} AND {{end_block}}
+                        WHERE block_number BETWEEN {{start_block}} AND {{end_block}}
```

Comment:
Just be careful with between since it uses a closed interval `block_number in [start, end]` while in other places we have `block_number in (start, end]`. Probably wouldn't hurt to eliminate all uses of BETWEEN.